docs/pages/includes/device-trust/prereqs.mdx

```diff
@@ -0,0 +1,4 @@
+- To enroll a macOS device, you need:
+  - A signed and notarized `tsh` binary for enrollment.
+    [Download the macOS tsh installer](../../installation.mdx#macos).
+  - A Teleport version newer than v12.0.0.
```

Comment:
"newer than v12.0.0" suggests that 12.0.0 will not work, and you need at least 12.0.1. Is that true?

Reply:
I will reword this as "A Teleport version v12.0.0 or above"